src/hypergraph.jl

```diff
@@ -295,4 +295,39 @@ function nhv(h::Hypergraph{T, V, E}) where {T <: Real, V, E}
     length(h.v2he)
 end
 
+"""
+    _walk!(h::Hypergraph, s::Vector{Int}, i::Vector{Int}, visited::Vector{Bool})
+
+Return the list of neighbors of a given vertex (an auxiliary function for get_connected_components).
```

Comment:
Enclose Julia commands in backticks
```suggestion
Return the list of neighbors of a given vertex (an auxiliary function for `get_connected_components`).
```